### ISSUE TITLE
Deployment resolves service but "svc show" doesn't

### ISSUE THREAD

**Issue by VaZark** [aws/copilot-cli#5151]
After verifying that only service is running on the console, on running `env show`, I get the following

```
Only found one service, defaulting to: cookie-video-banner
✘ describe service cookie-video-banner: retrieve service connect DNS names: get service cookie-video-banner: more than one
ECS service with the name cookie-video-banner found in environment test
```

However `copilot deploy` seems to identify the service and  deploy the service properly.  So does `copilot svc ls`

```
> copilot svc ls                                                                                                       
Name                 Type
----                 ----
cookie-video-banner  Load Balanced Web Service
```

**Comment by KollaAdithya** [comment]
Hello @VaZark!
Thanks for reporting the bug! we are doing a patch release `v1.29.1` soon. Will post the update here once it is out.

**Comment by KollaAdithya** [comment]
Hey @VaZark ! FIx is released in `v1.29.1` https://github.com/aws/copilot-cli/releases/tag/v1.29.1

**Comment by VaZark** [comment]
That was FAST! :) 

Thanks for the update
